### PR TITLE
(MAINT) Update CHANGELOG for 1.1.1 release

### PR DESCRIPTION
**Pull request by nwolfe**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 1.1.1
+
+* [TK-82](https://tickets.puppetlabs.com/browse/TK-82)
+  Add configuration option to control maximum number of
+  open HTTP connections that Jetty will maintain.
+* Upgrade Trapperkeeper dependency to 1.0.1.
+* Upgrade jvm-certificate-authority to 0.7.0.
```
Comment (camlow325) [inline CHANGELOG.md]: Maybe say "... to jvm-ssl-utils 0.7.0"?

**Comment by camlow325** [comment]
:+1: with minor comment.

**Comment by nwolfe** [comment]
@camlow325 updated inline

**Comment by camlow325** [comment]
:+1:

**Pull request merged**